jobs/env/pull-kubernetes-kubemark-e2e-gce.env

```diff
@@ -9,3 +9,5 @@ NODE_SIZE=n1-standard-2
 KUBELET_TEST_LOG_LEVEL=--v=4
 KUBEPROXY_TEST_LOG_LEVEL=--v=4
 ENABLE_HOLLOW_NODE_LOGS=true
+
+CREATE_CUSTOM_NETWORK=true
```

Comment:
I think this should be under `# cluster-env` section above.

Reply:
Thanks, fixed now.